specification/elasticsan/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the storagepool.
 
 ``` yaml
 openapi-type: arm
-tag: package-2024-05
+tag: package-2024-07-01-preview
```

Comment:
You should add a section for tag "package-2024-07-01-preview" in this file below.